back/src/database/UserSQL.js

```diff
@@ -34,22 +34,17 @@ async function getUserByEmail(email) {
     const query = {
         text: `SELECT * FROM users.users WHERE email='${email}'`
     };
-    return connection.connect().then(()=>{
-        return connection.query(query).then( result => {
-            connection.end();
-            if (result.rowCount == 0)
-                return {"success":false, "data":"User does not exist"};
-            info(fileLabel,"get user by email: " + util.inspect(email,{showHidden: false, depth: null}));
-            return {"success":true, "data":result.rows};
-        }).catch((exception)=>{
-            connection.end();
-            error(fileLabel,"Error while getting user. " + exception);
-            return {"success":false, "data":exception};      
-        });
+    const client = await pool.connect();
+    return client.query(query).then( result => {
+       client.release();
+        if (result.rowCount == 0)
+            return {"success":false, "data":"User does not exist"};
+        info(fileLabel,"get user by email: " + util.inspect(email,{showHidden: false, depth: null}));
```

Comment:
You can probably just put + email here instead of + util.inspect(email)...

You only need util when you want to look inside an object. In this case, email is just a string.